scenarios/crowdsecurity/http-generic-bf.yaml

```diff
@@ -0,0 +1,15 @@
+# 404 scan
```

Comment:
do we want to add this into the generic http collection ?

Reply:
not yet since it will not work with nginx currently

Reply:
Are there plans on when that might be? It's nice that crowdsec is detecting those now but since they're not linked to a scenario, it's just detected but not remediated.